docs/tutorials/basic/web.md

```diff
@@ -0,0 +1,219 @@
+---
+bodyclass: docs
+layout: docs
+headline: gRPC Basics - Web
+sidenav: doc-side-tutorial-nav.html
+type: markdown
+---
+<p class="lead">This tutorial provides a basic introduction on how to use
+gRPC-Web from browsers.</p>
+
+By walking through this example you'll learn how to:
+
+- Define a service in a .proto file.
+- Generate client code using the protocol buffer compiler.
+- Use the gRPC-Web API to write a simple client for your service.
+
+It assumes a passing familiarity with [protocol
+buffers](https://developers.google.com/protocol-buffers/docs/overview).
+
+<div id="toc"></div>
+
+<a name="why-grpc"></a>
+
+## Why use gRPC?
+
+With gRPC you can define your service once in a .proto file and implement
+clients and servers in any of gRPC's supported languages, which in turn can be
+run in environments ranging from servers inside Google to your own tablet - all
+the complexity of communication between different languages and environments is
+handled for you by gRPC. You also get all the advantages of working with
+protocol buffers, including efficient serialization, a simple IDL, and easy
+interface updating.
```

Comment:
Consider adding the line at the end:
gRPC-Web lets you access gRPC services built in this manner from browsers using an idiomatic API.

Reply:
Fixed